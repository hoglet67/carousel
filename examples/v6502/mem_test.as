        .ORG $8000

        ;; Fill first 256 bytes of memory with the number of each byte

MEM_TEST:
        LDA #$00
        TAY
        LDA #$00
        TAX
        STA $00
        
LOOP:   
        INY
        INX
        STY $0000,X
        BNE LOOP


END:    NOP
        JMP END

   