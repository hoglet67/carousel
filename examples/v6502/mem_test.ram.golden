  
0000: 00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f   ................
0010: 10 11 12 13 14 15 ff ff ff ff ff ff ff ff ff ff   ................
0020: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0030: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0040: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0050: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0060: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0070: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
